client/src/pages/Home.js

```diff
@@ -1,13 +1,12 @@
 import "./Home.css";
 import User from "../component/User";
-import MilestoneTable from  "../component/MilestoneTable";
+import Result from "../pages/Result"
 
 export function Home() {
-
 	return (
 		<main className="main_container" role="main">
 			<User />
-			<MilestoneTable />
```

Comment:
I am not sure 100% but I think we supposed to have the result on new pages as you created as a page in App.js and give it a path , and I know we were going to discuss this to be sure if its what we are going to do , but overall regarding the code you write in App.js I think the result Component shouldn't be here

Reply:
Yes, You are right, maybe in Search.js so when we clicked on search, the Result page will appear. The search component is not my Isshue so tried to don't code on it to avoid conflict